packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -161,6 +161,25 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       }),
     },
 
+    blobs: {
+      getBlobsV1Requests: register.gauge({
+        name: "beacon_get_blobs_v1_requests_total",
+        help: "Number of getBlobsV1 requests that get made",
+      }),
+      getBlobsV1Error: register.gauge({
+        name: "beacon_get_blobs_v1_response_error_total",
+        help: "Number of getBlobsV1 calls that errored ",
+      }),
+      getBlobsV1Miss: register.gauge({
+        name: "beacon_get_blobs_v1_missing_blob_response_total",
+        help: "Number of getBlobsV1 misses where a versioned hash returns a null",
+      }),
+      getBlobsV1Hit: register.gauge({
+        name: "beacon_get_blobs_v1_blob_returned_response_total",
+        help: "Number of getBlobsV1 hits where a versioned hash returns blob",
+      }),
+    },
```

Comment:
I'd rather keep existing metrics and re-wire them in new code since we already have dashboards for those. 

These metrics are also missing how many blobs were actually useful for us which was `beacon_datapromise_blockinput_getblobs_api_nonnull_responses_used_total` previously